### ISSUE TITLE
Frontend Buchungsdetailseite (unconfirmed) umsetzen: 

### ISSUE THREAD

**Issue by hanneskasimir** [wielebenwir/commons-booking-2#537]
https://miro.com/app/board/o9J_kyepl0c=/?moveToWidget=3074457347182694146

**Comment by flegfleg** [comment]
> Links zu Item und Location existieren immer noch.
> Überschrift Name immer noch da.
> Neu: Bitte Mailadresse auch nicht verlinken.

Können wir das zurückstellen? Es nimmt das selbe Template wie die Liste, würde da ohne dass wir gemeinsam noch mal konzeptionell drüber geschaut haben erst mal keine neuen Templates bauen 

> Eintrag neben Name fehlt.

dann hast du keinen Namen  eingegeben :)

**Comment by hanneskasimir** [comment]
Ja, unbedingt zurückstellen. Deshalb war es auch "next" und nicht bei "todo" ;-)
